skimage/__init__.py

```diff
@@ -159,3 +160,7 @@ def _raise_build_error(e):
 
 
 del warnings, functools, osp, imp, sys
+
+
+def lookfor(search_text):
+    return np.lookfor(search_text, sys.modules[__name__])
```

Comment:
Alternatively, the function could be defined elsewhere such as:
skimage/_shared/utils.py
and then imported from there